doc/nrf/releases_and_maturity/releases/release-notes-2.6.0.rst

```diff
@@ -0,0 +1,1393 @@
+    * Renamed :file:`overlay-pdn_ipv4.conf` to :file:`overlay-pdn-nrf91-ipv4.conf` and :file:`overlay-tfm_mbedtls.conf` to :file:`overlay-tfm-nrf91.conf`.
+
+* :ref:`download_sample` sample:
+
+  * Updated:
+
+    * The format of the :file:`.pem` file to the PEM format.
+    * The sample to automatically convert the :file:`.pem` file to HEX format so it can be included.
+    * The sample to gracefully bring down the network interfaces.
+
+
+nRF5340 samples
+---------------
+
+* Added the :ref:`smp_svr_ext_xip` sample that demonstrates how to split an application that uses internal flash and Quad Serial Peripheral Interface (QSPI) flash with the Simple Management Protocol (SMP) server.
+
+Peripheral samples
+------------------
+
+* :ref:`radio_test` sample:
+
+  * Updated:
+
+    * The ``start_tx_modulated_carrier`` command, when used without an additional parameter, does not enable the radio end interrupt.
+    * Corrected the way of setting the TX power with FEM.
+
+PMIC samples
+------------
+
+* Added :ref:`npm1300_one_button` sample that demonstrates how to support wake-up, shutdown and user interactions through a single button connected to the nPM1300.
+
+* :ref:`npm1300_fuel_gauge` sample:
+
+  * Updated to accommodate API changes in the :ref:`nrfxlib:nrf_fuel_gauge`.
+
+Thread samples
+--------------
+
+* Added experimental support for Thread Over Authenticated TLS.
+* Updated the building method to use :ref:`zephyr:snippets` for predefined configuration.
+
+* Removed:
+
+  * In the :ref:`thread_ug_feature_sets` provided as part of the |NCS|, the following features have been removed from the FTD and MTD variants:
+
+    * ``DHCP6_CLIENT``
+    * ``JOINER``
+    * ``SNTP_CLIENT``
+    * ``LINK_METRICS_INITIATOR``
+
+    All mentioned features are still available in the master variant.
+
+Wi-Fi samples
+-------------
+
+* Added:
+
+  * The :ref:`wifi_throughput_sample` sample that demonstrates how to measure the network throughput of a Nordic Semiconductor Wi-Fi enabled platform under the different Wi-Fi stack configuration profiles.
+  * The :ref:`wifi_softap_sample` sample that demonstrates how to start a nRF70 Series device in :term:`Software-enabled Access Point (SoftAP or SAP)` mode.
+  * The :ref:`wifi_raw_tx_packet_sample` sample that demonstrates how to transmit raw TX packets.
+  * The :ref:`wifi_monitor_sample` sample that demonstrates how to start a nRF70 Series device in :term:`Monitor` mode.
+
+* :ref:`wifi_shell_sample` sample:
+
+  * Updated the sample by adding the following extensions to the Wi-Fi command line:
+
+    * ``raw_tx`` extension.
+      It adds the subcommands to configure and sends raw TX packets.
+
+    * ``promiscuous_set`` extension.
+      It adds the subcommand to configure Promiscuous mode.
+
+* :ref:`wifi_ble_coex_sample` sample:
+
+  * Updated the sample folder name from ``sr_coex`` to ``ble_coex`` to accurately represent the functionality of the sample.
+
```

Comment:
PR https://github.com/nrfconnect/sdk-nrf/pull/14410 is merged in with a changelog entry to release-notes-changelog.rst file. It should be added here as:
```suggestion

* :ref:`wifi_radio_test` sample:

  * Updated:

    * Added support for runtime configuration of antenna gain and edge backoff parameters.
    * Antenna gain and edge backoff values are not applied if regulatory is bypassed.
```